sampleConfigurations/videojs/videojs/configuration.dsc.yaml

```diff
@@ -1,5 +1,5 @@
 # yaml-language-server: $schema=https://aka.ms/configuration-dsc-schema/0.2
-# Reference: https://github.com/videojs/video.js/blob/main/CONTRIBUTING.md#building-videojs-locally
+# Reference: https://github.com/videojs/videojs/blob/main/CONTRIBUTING.md#building-videojs-locally

```

Comment:
Does this need to be changed? This link becomes invalid if you remove the '.' in video.js. If that is causing issues you can remove this comment entirely.

Reply:
Fixed.